elastica/boundary_conditions.py

```diff
@@ -294,6 +299,12 @@ def __init__(self, *fixed_data, **kwargs):
             Tuple of position-indices that will be constrained
         constrained_director_idx : tuple
             Tuple of director-indices that will be constrained
+        translational_constraint_selector: np.array = np.array([True, True, True])
```

Comment:
```suggestion
        translational_constraint_selector: Optional[np.ndarray] = None
```

Reply:
Default numpy value can be set within `__init__` funciton.

Reply:
Fixed in https://github.com/GazzolaLab/PyElastica/pull/143/commits/03fc1502110db67904ceb26648b378d982e6cdf1